Tests/ApolloInternalTestHelpers/MockOperation.swift

```diff
@@ -84,7 +84,7 @@ open class MockFragment: MockSelectionSet, Fragment {
   open class var fragmentDefinition: StaticString { "" }
 }
 
-open class MockTypeCase: MockSelectionSet, InlineFragment {
+open class MockTypeCase: MockSelectionSet, InlineFragment, Deferrable {
```

Comment:
I might change the composition of `InlineFragment` to include `Deferrable` in a later PR. `Fragment` conforms to `Deferrable` so it makes sense that `InlineFragment` should too. It would clean up the codegen and generated models a bit too.